bin/v-import-cpanel

```diff
@@ -285,6 +273,46 @@ else
 	tput sgr0
 fi
 
+# Parked domains
+if [ -s pds ]; then
+    cat pds | while read parked
+    do
+        echo "Procesing parked domain: $parked"
+        parkedfor=$(cat userdata/cache.json |  jq --arg domain "$parked" '.[$domain][3]' |sed 's/"//g')
+        $BIN/v-add-web-domain-alias $new_user $parkedfor $parked
+    done
+else
+
+    echo "No parked domains found"
+
+fi
+
+# Try SSL
+tput setaf 2
+    echo "Copy SSL files"
+tput sgr0
+for ssl_domain in apache_tls/*
+do
+    domain=$(echo $ssl_domain | awk -F '/' '{ print $2 }')
+    mkdir -p apache_tls/ssl/$domain
+    awk -v RS="-----BEGIN CERTIFICATE-----" -v ssl_domain="$domain"  '
```

Comment:
This is supposed to split a single file into cert key, ssl certificate and ca certificate if I understand this correctly - However, With NR=1, it can be anything before the first time -----BEGIN CERTIFICATE-----, and it doesn't actually verify it's a certificate key. 

Similarly, It doesn't verify if the other content is actually the certificates we wanted. 

also, for ssl_domain in apache_tls/* will also run through a domain called ssl after the first loop where it has been created by the mkdir -p apache_tls/ssl/ command. 

Something like this could be a start: 
```

for ssl_domain in apache_tls/*
do
        if [ -f $ssl_domain ]; then
                domain=$(echo $ssl_domain | awk -F '/' '{ print $2 }')
                mkdir -p apache_tls/ssl/$domain
                # Extract certificate key
                openssl pkey -in $ssl_domain -out apache_tls/ssl/$domain/$domain.key
                #Extract all certificates from original file
                openssl crl2pkcs7 -nocrl -certfile $ssl_domain | openssl pkcs7 -print_certs -out $(echo $domain)_all.pem
                cat $(echo $domain)_all.pem | awk -v ssl_domain="$domain" '/BEGIN/ { i++; } /BEGIN/, /END/ { print > i "." ssl_domain "_all.crt" }'
                ls *.$(echo $domain)_all.crt | while read cert; do if [ $(openssl x509 -noout -subject -in $cert | sed -n 's/.*CN = \(.*\)/\1/p') == $(echo $domain) ] ; then cat $cert > apache_tls/ssl/$(echo $domain)/$(echo $domain).crt; else cat $cert > apache_tls/ssl/$(echo $domain)/$(echo $domain).ca; fi ; done
                rm *.$(echo $domain)_all.crt
                rm $(echo $domain)_all.pem
        fi
done
```

Reply:
Feel free to modify this in any better way, I  have some problems with this with diferent ssl and not found any good way to handle this when certificate file have more then 1 CA, not research to much, all people use now lets encrypt is easy just to install new certificate.